spec/features/users_can_view_project_information_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.feature "Users can view project information" do
+  let(:user) { create(:user, email: "user@education.gov.uk") }
+  let(:project) { create(:project, delivery_officer: user) }
+  let(:project_id) { project.id }
+
+  before do
+    mock_successful_api_responses(urn: 12345)
+    sign_in_with_user(user)
+  end
+
+  scenario "User views project information" do
+    visit project_information_path(project_id)
+
+    expect(page).to have_content("Project details")
+
+    delivery_officer_label = page.find("dt", text: "Delivery officer")
```

Comment:
When we add more things we want to test here it might be worth DRYing up these "find label and check its contents", but for now I'm happy with this!

Reply:
Sounds good! Thanks for the review 🚢